pom.xml

```diff
@@ -204,6 +196,10 @@
 					<header>${project.name}</header>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
Why do we need to duplicate this plugin?

Reply:
Where do we duplicate it?

Reply:
One is in the reporting section and the other is in the main one

Reply:
Right. I understand the layout. Only my concern that I don't see answer to the question "Why?".

> 5. Add checkstyle plugin to your build and reporting phases

Is not a reason to do this.
Why that reporting can't get appropriate plugins from the main section?

However what I see, according the plugin official docs, we just don't need it in the main section: https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html

That's how I am 🤷‍♂️ 

Thanks for understanding.

Reply:
Maybe I misconfigured it then. If you think that it's wrong then let's change it.